doc/api/webstreams.md

```diff
@@ -492,7 +492,7 @@ added: v16.5.0
 -->
 
 * Returns: A promise fulfilled with an object:
-  * `value` {ArrayBuffer}
+  * `value` {any}
```

Comment:
```suggestion
  * `value` {unknown}
```

Reply:
I don't see `{unknown}` being used anywhere in `docs/api/`, only `{any}`... 😕